sdk/clientcore/tools/annotation-processor-dependency-tests/src/main/java/io/clientcore/tools/codegen/MockHttpResponse.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.tools.codegen;
```

Comment:
Remove `tools` from the package name as we don't have it in the package groupId now.

Reply:
We should follow the convention of using group id followed by artifact id for package names - `io.clientcore.annotation.processor.dependency.test`.